<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<HTML>
 <HEAD>
   <TITLE> [mod_python] mod_python compiles fine, but gives errors...
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20mod_python%20compiles%20fine%2C%20but%20gives%20errors...&In-Reply-To=">
   <META NAME="robots" CONTENT="index,nofollow">
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="000448.html">
   <LINK REL="Next"  HREF="000450.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[mod_python] mod_python compiles fine, but gives errors...</H1>
    <B>Ray Van Dolson</B> 
    <A HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20mod_python%20compiles%20fine%2C%20but%20gives%20errors...&In-Reply-To="
       TITLE="[mod_python] mod_python compiles fine, but gives errors...">rayvd at firetail.org
       </A><BR>
    <I>Thu Dec  7 12:04:27 EST 2000</I>
    <P><UL>
        <LI>Previous message: <A HREF="000448.html">[mod_python] How do I find the document root?
</A></li>
        <LI>Next message: <A HREF="000450.html">[mod_python] python_handler: Dispatch() returned nothing.
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#449">[ date ]</a>
              <a href="thread.html#449">[ thread ]</a>
              <a href="subject.html#449">[ subject ]</a>
              <a href="author.html#449">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>I'm compiling mod_python (latest version) as DSO into Apache 1.3.12
against a Python source tree with no threads.  Everything compiles fine
and even seems to install fine.  However, when starting up apache, it
gives me the following error:

[Thu Dec  7 11:58:16 2000] [warn] Loaded DSO libexec/mod_python.so usesplain Apache 1.3 API, this module might crash under EAPI! (please recompile it with -DEAPI)
[Thu Dec  7 11:58:16 2000] [notice] Apache/1.3.12 (Unix) mod_ssl/2.6.2 OpenSSL/0.9.4 mod_python/2.6.3 Python/1.5.2 mod_perl/1.22 PHP/3.0.16 configured -- resuming normal operations

What is this EAPI warning?  Should I worry about it?  Is it related in any
way to the following error I get when running the test program?  (This is
from the error_log file):

make_obcallback(): could not import mod_python.apache.
make_obcallback(): could not call init.
[Thu Dec  7 11:58:35 2000] [error] [client 199.236.177.11] python_handler: make_obcallback returned no obCallBack!

Apache is running as user nobody, which I read might pose a problem.  But
all the Python modules should be readable by this user.

Any ideas?

Ray Van Dolson


</PRE>
<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="000448.html">[mod_python] How do I find the document root?
</A></li>
	<LI>Next message: <A HREF="000450.html">[mod_python] python_handler: Dispatch() returned nothing.
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#449">[ date ]</a>
              <a href="thread.html#449">[ thread ]</a>
              <a href="subject.html#449">[ subject ]</a>
              <a href="author.html#449">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="http://mailman.modpython.org/mailman/listinfo/mod_python">More information about the Mod_python
mailing list</a><br>
</body></html>
